cloud/pkg/router/utils/path_test.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestURLToURLRegex(t *testing.T) {
+	uri := "/aaa/bbb/ccc/ddd"
+	uriReg := URLToURLRegex(uri)
+	AssertStringEqual(t, uri+"/{0,1}", uriReg, "1")
+
+	uri = "/aaa/bbb/{sssss}/ddd"
+	uriReg = URLToURLRegex(uri)
+	AssertStringEqual(t, "/aaa/bbb/[-A-Za-z0-9+&@#%?=~_|!:,.;]+/ddd/{0,1}", uriReg, "2")
```

Comment:
just refer pathRegex for `[-A-Za-z0-9+&@#%?=~_|!:,.;]+`, and define and refer tailSlash for `/{0,1}`.

Reply:
done